.flowconfig

```diff
@@ -5,6 +5,8 @@
 [options]
 module.system=node
 log.file=./artifacts/flow.log
+suppress_comment= \\(.\\|\n\\)*\\$FLOW_FIXME
```

Comment:
What is the first part of this pattern matching? Could we keep it simple and make ``// \$FLOW_FIXME` ?